pkg/project/cloud/Taskfile.yml

```diff
@@ -118,27 +121,29 @@ tasks:
         echo "$PROPS" | sh aemw repl agent setup -P --location "publish" --name "flush"
 
   aem:provision:crx:
-    desc: enable CRX/DE
+    desc: enables CRX/DE on all AEM instances
+    internal: true
     cmds:
       - 'sh aemw osgi config save --pid "org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet" --input-string "alias: /crx/server"'
 
   aem:provision:apm:
-    desc: deploy APM tool
+    desc: deploys APM tool
+    internal: true
     cmds:
       - sh aemw package deploy --url "https://github.com/wttech/APM/releases/download/apm-{{.APM_VERSION}}/apm-all-{{.APM_VERSION}}.zip"
 
   aem:destroy:
-    desc: destroy AEM instances
+    desc: destroys all AEM instances
     deps: [aem:stop]
     cmds: [sh aemw instance destroy]
 
   aem:stop:
-    desc: stop AEM instances
+    desc: stops AEM instances
     aliases: [aem:down]
     cmds: [sh aemw instance stop]
 
   aem:build:
-    desc: build AEM application
+    desc: builds AEM application in current folder using Maven
```

Comment:
what does it mean in current folder ?:D I don't get it why it is important to mention it here